src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -1,16 +1,47 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <!-- Include common build properties -->
+<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+
   <!-- Compilation options -->
+  <Import Project="../../clr.coreclr.props" />
+  <Import Project="../../clr.defines.targets" />
+
+  <!-- Generate rsp -->
+  <Import Project="GenerateCompilerResponseFile.targets" />
+
+  <Target Name="GenerateTheRSPFile" BeforeTargets="CoreCompile" DependsOnTargets="GenerateCompilerResponseFile">
+  </Target>
+
+  <PropertyGroup>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <GenerateResxSourceOmitGetResourceString>true</GenerateResxSourceOmitGetResourceString>
+    <EnsureRuntimePackageDependencies>false</EnsureRuntimePackageDependencies>
+
+    <OutputType>Library</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+
+    <!-- Default is embedded overwrite this -->
+    <DebugType>Portable</DebugType>
+
+    <!-- Force System.Private.CoreLib.dll into a special IL output directory -->
+    <OutputPath>$(BinDir)IL/</OutputPath>
+  </PropertyGroup>
+
+  <!-- Create SR.cs -->
```

Comment:
This was added to allow SR.cs to be created with Arcade. I would suggest not making this change without the arcade changes. It is worth in my opinion adding a condition on ArcadeBuild. If we decide to keep the change, SPC should be inspected to make sure the generated embedded resource is equivalent to before the change.